.github/workflows/kind-e2e.yaml

```diff
@@ -84,9 +84,9 @@ jobs:
       run: |
         set -o pipefail
 
-        IMAGE=$(ko publish ./test)
+        IMAGE=$(ko build ./test)
         SBOM=$(cosign download sbom ${IMAGE})
-        KO_DEPS=$(ko deps ${IMAGE})
+        KO_DEPS=$(ko deps --sbom=spdx ${IMAGE})
```

Comment:
Can you make this the default for `ko deps` too?

Reply:
Sure